plugin/mmdb/README.md

```diff
@@ -0,0 +1,56 @@
+# mmdb
+## Name
+*mmdb* - Lookup maxmind geoip2 databases then add associated data to the context request.
+
+## Description
+*mmdb* - Allows you to configure [geoIP2 maxmind databases](https://dev.maxmind.com/geoip/docs/databases) and automatically associate and DNS request data matching the client IP.
+
+Common use cases would be adding per request GEO location data, ISP, ASN, etc.
+
+The data will be added to the request context leveraging the [metadata](../metadata/README.md) plugin, and therefore data will be available by make use of it, label names will be created using the following convention.
+
+## Databases
+The following official [max mind databases](https://dev.maxmind.com/geoip/docs/databases) are allowed.
+
+- Anonymous IP
+- ASN
+- City
+- ConnectionType
+- Country
+- Domain
+- Enterprise
+- ISP
+
+Databases with custom schemas are not allowed, but may be supported in the future.
+
+## Metadatada Label/Value Convention
+**Label**: plugin/mmdb/`KEYNAME`/...
+
+**Value**: Will be the associated field value in its string representation. 
+
+> Note: Numberic types will be represented in base 10.
+
+For example, in a `City` database but not limited to those fields:
+
+| Label                                                 | Value            |
+| :---------------------------------------------------- | :-------------   |
+| `plugin/mmdb/city/names/en`                           | `Cambridge`      |
+| `plugin/mmdb/country/names/en`                        | `United Kingdom` |
+| `plugin/mmdb/continent/names/en`                      | `Europe`         |
+| `plugin/mmdb/registered_country/is_in_european_union` | `false`          |
+| `plugin/mmdb/location/longitude`                      | `0.1315`         |
+| `plugin/mmdb/location/latitude`                       | `52.2242`        |
+
+## Syntax
+```txt
+mmdb [DBFILE]
```

Comment:
```suggestion
mmdb DBFILE [DBFILE]...
```

Reply:
I was thinking that path are potentially long, so instead of providing them all into a single `mmdb` statement, users could provide one path into one `mmdb` statement, but I'm happy to do as you suggest if that's the preferred way.

Reply:
Plugin config block changed to
```
geoip [DBFILE] {
    languages [CODE]...
}
```